website/pages/docs/plugins/destinations/overview.mdx

```diff
@@ -3,32 +3,28 @@ title: Destinations
 ---
 
 import { PluginsTable } from "../../../../components/PluginsTable";
+import { getPluginsData } from '../../../../utils/plugins'
+import { useSSG } from 'nextra/ssg'
+
+{/* <!-- vale off --> */}
+export const getStaticProps = () => {
```

Comment:
We need `getStaticProps` as `getPluginsData` uses `fs` so we need to tell Next.js to run it on "Server" context. This should run only once during build as we don't revalidate it